app.js

```diff
@@ -21,11 +21,12 @@ exports.connect = function (formattedTime) {
   })
 
   // authenticate with the GitHub API, if possible
-  if (process.env.GITHUB_OAUTH) {
+  if (process.env.GITHUB_OAUTH !== undefined) {
```

Comment:
Could we make this `if (typeof process.env.GITHUB_OAUTH !== 'undefined')`?

[This StackOverflow answer explains it well](https://stackoverflow.com/a/4725697) 👍